SassHandler.php

```diff
@@ -203,15 +203,14 @@ class SassHandler extends CApplicationComponent
      */
     public function init()
     {
-        $vendorPath = __DIR__ . '/../../';

```

Comment:
The old `$vendorPath` is correct for installations using Composer.
Could you please explain your change?